test/e2e/deploy/deploy.go

```diff
@@ -758,7 +791,7 @@ func (d *Deployment) GetDriverDeployment() api.Deployment {
 			Labels: map[string]string{
 				deploymentLabel: d.Name,
 			},
-			// TODO: replace pmemcsidriver.DeviceMode with api.DeviceMode everywhere
+			// TODO: replace pmemcsidrive1r.DeviceMode with api.DeviceMode everywhere
```

Comment:
Can you please remove this unintentional change.

Reply:
Fixed.